### DISCUSSION TITLE
Support for special characters

### DISCUSSION THREAD

**Discussion by hitesh7424** [vedicscript/vedicscript#13] [Ideas]
I love the language just one suggestion(many will come in future) that pls add support for special character like underscore '_' or other you like for naming variables (मान) as all letters are equal size
example: एचटीटीपी_स्थितिसंकेतः it is more readable than एचटीटीपीस्थितिसंकेतः

**Comment by PtPrashantTripathi** [comment]
Hi @hitesh7424,

Thanks for your suggestion, v2 is currently in the development phase i'll Keep your suggestion in mind.

शुभ रात्रि
